tests/integration_tests/performance/test_block_ab.py

```diff
@@ -93,21 +86,25 @@ def run_fio(microvm, mode, block_size, fio_engine="libaio"):
         )
 
         # Print the fio command in the log and run it
-        rc, _, stderr = microvm.ssh.run(f"cd /tmp; {cmd}")
+        rc, stdout, stderr = microvm.ssh.run(f"cd /tmp; {cmd}")
         assert rc == 0, stderr
         assert stderr == ""
 
-        microvm.ssh.scp_get("/tmp/*.log", logs_path)
+        fio_json_path = Path(test_output_dir / "fio.json")
+        with open(fio_json_path, "w", encoding="utf-8") as f:
+            f.write(stdout)
+
+        microvm.ssh.scp_get("/tmp/*.log", test_output_dir)
         microvm.ssh.check_output("rm /tmp/*.log")
 
-        return logs_path, cpu_load_future.result()
+        return cpu_load_future.result()
 
 
-def process_fio_log_files(logs_glob):
+def process_fio_log_files(root_dir, logs_glob):
     """Parses all fio log files matching the given glob and yields tuples of same-timestamp read and write metrics"""
     data = [
-        Path(pathname).read_text("UTF-8").splitlines()
-        for pathname in glob.glob(logs_glob)
+        Path(root_dir / pathname).read_text("UTF-8").splitlines()
+        for pathname in glob.glob(logs_glob, root_dir=root_dir)
```

Comment:
I admittedly don't quite see why this change to this function is needed?

Reply:
Ah, fair. Let's put that in the commit message/a comment, otherwise I can see it getting "simplified" again in the future by someone not aware of the context

Reply:
(and let's also validate this assumption is actually true before changing the code)

Reply:
checked that this is indeed the case and added a comment

Reply:
thanks!